packages/libs/react-ui/src/Intro.mdx

```diff
@@ -0,0 +1,281 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta
+  title="Welcome/Intro"
+  parameters={{ options: { isToolshown: false }, id: 1 }}
```

Comment:
The Intro and Contributing files are complete duplicates of existing md files. Can we just reference those instead and add this meta data there?

Reply:
Yeah true, that would make more sense. I'll have a look!